ec/acc/deployment_config_test.go

```diff
@@ -15,9 +15,153 @@
 // specific language governing permissions and limitations
 // under the License.
 
+// +build acceptance
+
 package acc
 
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/elastic/cloud-sdk-go/pkg/api/deploymentapi/deptemplateapi"
+	"github.com/elastic/cloud-sdk-go/pkg/api/stackapi"
+	"github.com/elastic/cloud-sdk-go/pkg/models"
+)
+
 const (
-	deploymentVersion = "7.9.2"
-	region            = "us-east-1"
+	defaultTemplate = "io-optimized"
+	hotWarmTemplate = "hot-warm"
+
+	// This deployment template is only used for the
+	// TestAccDatasourceDeployment_basic test.
+	depsDSTemplate = "compute-optimized"
 )
+
+func getRegion() string {
+	region := "us-east-1"
+
+	if r := os.Getenv("EC_REGION"); r != "" {
+		region = r
+	}
+
+	return region
+}
+
+func latestStackVersion() (string, error) {
+	client, err := newAPI()
+	if err != nil {
+		return "", err
+	}
+
+	res, err := stackapi.List(stackapi.ListParams{
+		API:    client,
+		Region: getRegion(),
+	})
+	if err != nil {
+		return "", err
+	}
+
+	return res.Stacks[0].Version, nil
+}
+
+func setDefaultTemplate(region, template string) string {
```

Comment:
I initially had `ece-region` return the default template as well, but we we use more than one template for the tests :( 
The `datasource` tests would still work I guess, but the hot/warm tests would cause trouble 🤔 Also, we cannot guarantee the deployment template Ids in people's ECE installations :/

Reply:
alright, we can leave the ECE use case out for now anyway.